.github/workflows/deploy_monitoring_dev.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           TF_VAR_google_client_id: ${{ secrets.GOOGLE_CLIENT_ID }}
           TF_VAR_google_client_secret: ${{ secrets.GOOGLE_CLIENT_SECRET }}
           TF_VAR_grafana_admin_password: ${{ secrets.GRAFANA_ADMIN_PASSWORD_DEV }}
+          TF_VAR_alertmanager_slack_url: ${{ secrets.ALERTMANAGER_SLACK_URL }}
```

Comment:
holy crap, will we get slack notifications when something goes wrong? :O

Reply:
@kicferk1 Yeah we're going to DM you directly at 2am :trollface: